### ISSUE TITLE
Why are there two works for Gelasius? tlg2768.tlg001 and tlg2768.002

### ISSUE THREAD

**Issue by AlisonBabeu** [OpenGreekAndLatin/First1KGreek#1494]
Hi @sonofmun and @annettegessner, this isn't an issue I suppose so much as a question.
I just opened the Gelasius files and see that there are two separate works.

The first is tlg2768.tlg001.opp-grc1 [Historia Ecclesiastica](https://github.com/OpenGreekAndLatin/First1KGreek/blob/master/data/tlg2768/tlg001/tlg2768.tlg001.opp-grc1.xml) and everything seems in order.

The second tlg2768.tlg002.opp-grc1, and tlg2768.002.opp-lat1, is rather oddly titled, 
"According to Eusebius and Theognius", which appears to be an English translation of the Greek subtitle, and is found in pages 192-199 of the source edition (https://archive.org/stream/gelasiuskircheng28gelauoft#page/192/mode/2up). My main question is why has this section been separated out as an individual work? The TLG used this same edition for the Historia Ecclesiastica but did not separate out this last section as a separate work.

**Comment by sonofmun** [comment]
I think there are actually two different works in the Anhang, neither of which are from Gelasius. According to the note here (https://archive.org/stream/gelasiuskircheng28gelauoft#page/n41/mode/2up), they are two letter from Constantin, one to the Church in Nicomedien and one to Theodotian. The notes suggest that these are actually normally associated with Athanasius (tlg2035). I am not sure where to put them. But one thing seems to be certain: they are not part of Gelasius' Historia Ecclesiastica.

**Comment by sonofmun** [comment]
BTW, it looks like the κατα should mean "against" here and not "according to". So the first is a letter from Constantine "Against Eusebius (of Nicomedia) and Theognis". To deal with these correctly we may need to use a CITE URN since this is a work that is quoted in these texts but doesn't actually exist.

**Comment by AlisonBabeu** [comment]
So @sonofmun  just to check back on this, I did a search of the text in the TLG and it looks like both of these two Anhangs/Appendices are also included in the canon as part of the work of Athanasius, tlg2035.tlg003, De decretis Nicaenae synodi, chapters 41 and 42. Should we move this "second" work by Gelasius to a new directory under Athanasius (an author for whom we suspiciously have no works.....) and then create new URNs?

**Comment by sonofmun** [comment]
Yes, let's do this. Thanks for taking the time to figure this out, @AlisonBabeu

**Comment by AlisonBabeu** [comment]
@sonofmun I think this one can be closed now yes?
